### PR TITLE
Added | News | Introducing the Apple Developer App

### PR DESCRIPTION
**Pull request by irshadpc**
Added | News | Introducing the Apple Developer App

### CODE REVIEW

File: News/306-ApplelaunchedTheDeveloperApp.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "title": "Introducing the Apple Developer App",
+    "description": " the primary destination for Apple developers and designers. Apple Developer is your source for developer news, educational content, the WWDC experience, and more, so you can stay up to date on the latest technical and community information",
+    "url": "https://apps.apple.com/in/app/apple-developer/id640199958",
```
Comment (pedrommcarrasco) [inline News/306-ApplelaunchedTheDeveloperApp.json]: Let's remove some of the boilerplate here:
`https://apps.apple.com/app/id640199958`

File: News/306-ApplelaunchedTheDeveloperApp.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "curator": {
+        "name": "Irshad PC",
```
Comment (pedrommcarrasco) [inline News/306-ApplelaunchedTheDeveloperApp.json]: 💪

File: News/306-ApplelaunchedTheDeveloperApp.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "title": "Introducing the Apple Developer App",
```
Comment (pedrommcarrasco) [inline News/306-ApplelaunchedTheDeveloperApp.json]: What do you think of:
`Apple launches app for developers`

File: News/306-ApplelaunchedTheDeveloperApp.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    "description": " the primary destination for Apple developers and designers. Apple Developer is your source for developer news, educational content, the WWDC experience, and more, so you can stay up to date on the latest technical and community information",
```
Comment (pedrommcarrasco) [inline News/306-ApplelaunchedTheDeveloperApp.json]: What about:
`What was once the WWDC app is now the Apple Developer app and it delivers the latest developer news, informative videos, WWDC content and more.`

**Comment by irshadpc** [comment]
> Love this initiative by Apple! ❤️
> Left some comments here and there 😄

 😄

**Pull request merged**